Inputs/inputs_2e2mu_2016.yml

```diff
@@ -56,17 +56,17 @@ Nuissances:
     Type: lnN
     Value: '0.756295/1.25114'
   106n:
-    Name: CMS_zz4l_mean_m_sig_2016
+    Name: CMS_zz4l_mean_m_sig
     Applied_To:
       - All
     Type: param 
     Value: '0.0 1.0'
   107n:
-    Name: CMS_zz4l_sigma_m_sig_2016
+    Name: CMS_zz4l_sigma_m_sig
     Applied_To:
       - All
     Type: param 
-    Value: '0.0 0.2 [-1,1]'
+    Value: '0.0 0.03 [-1,1]'
```

Comment:
@tjavaid  Why did you change the range?

Reply:
have moved Chengguang's studies:

https://indico.cern.ch/event/1154424/contributions/4862783/attachments/2439275/4178136/04052022_higgsmasswidth_leptonscaleandresolution.pdf